### PR TITLE
[Icons] Added new UI icons, removed unused "snippet" icons

### PR DESCRIPTION
**Pull request by pkwi**
Added new UI icons: Filter: keep, Filter: remove, Key, Feed: subscribed.

### CODE REVIEW

File: packages/core/resources/icons/icons.json [hunk 1]
```diff
@@ -2856,10 +2856,38 @@
     "content": "\\e78a"
   },
   {
-    "name": "New prescription",
+    "name": "Prescription: new",
     "className": "pt-icon-new-prescription",
```
Comment (cmslewis) [inline packages/core/resources/icons/icons.json]: This `className` is inconsistent with the `name` now. Do we want to track changing this to `pt-icon-prescription-new` (or maybe supporting both that and `pt-icon-new-prescription`)?

File: resources/icons/16px/pt-icon-snippet.svg [hunk 2]
```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (cmslewis) [inline resources/icons/16px/pt-icon-snippet.svg]: What is this `snippet` icon? I don't see it on blueprintjs.com right now.

File: packages/core/resources/icons/icons.json (hunk 1 above)
Comment (adidahiya) [reply packages/core/resources/icons/icons.json]: please don't change the class name or support multiple names. display names can diverge, that's ok

File: resources/icons/16px/pt-icon-snippet.svg (hunk 2 above)
Comment (giladgray) [reply resources/icons/16px/pt-icon-snippet.svg]: i think that's why @pkwi is removing it: it is actually unused.

Comment (giladgray) [reply resources/icons/16px/pt-icon-snippet.svg]: also it only appeared in 16px, not 20px

**Pull request merged**